<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Translater</string>
  </property>
  <property name="windowIcon">
   <iconset resource="main_ico.qrc">
    <normaloff>:/newPrefix/resources/translate.ico</normaloff>:/newPrefix/resources/translate.ico</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QGroupBox" name="groupBox">
      <property name="statusTip">
       <string>选择与中文相对的语种</string>
      </property>
      <property name="title">
       <string>译文语种</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QRadioButton" name="radio_en">
           <property name="text">
            <string>英语</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_ja">
           <property name="text">
            <string>日语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_ko">
           <property name="text">
            <string>韩语</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_ru">
           <property name="text">
            <string>俄语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_de">
           <property name="text">
            <string>德语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_fr">
           <property name="text">
            <string>法语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_es">
           <property name="text">
            <string>西班牙语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_pt">
           <property name="text">
            <string>葡萄牙语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_it">
           <property name="text">
            <string>意大利语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_ar">
           <property name="text">
            <string>阿拉伯语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_id">
           <property name="text">
            <string>印尼语</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radio_vi">
           <property name="text">
            <string>越南语</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>原文：</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>译文：</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QTextEdit" name="con_in">
      <property name="statusTip">
       <string>在此输入需要翻译的内容，支持单词、句子，段落。</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#000000</string>
      </property>
      <property name="acceptRichText">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPlainTextEdit" name="con_out">
      <property name="statusTip">
       <string>在此显示翻译结果</string>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#000000</string>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_fy">
          <property name="text">
           <string>翻译</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="con_in_size">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>设置原文文字大小</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:none;}</string>
          </property>
          <property name="icon">
           <iconset resource="main_ico.qrc">
            <normaloff>:/newPrefix/resources/font.png</normaloff>:/newPrefix/resources/font.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="con_in_color">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>设置原文文字颜色</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:none;}</string>
          </property>
          <property name="icon">
           <iconset resource="main_ico.qrc">
            <normaloff>:/newPrefix/resources/color.png</normaloff>:/newPrefix/resources/color.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>10</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>10</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="con_out_color">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>设置译文文字颜色</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:none;}</string>
          </property>
          <property name="icon">
           <iconset resource="main_ico.qrc">
            <normaloff>:/newPrefix/resources/color.png</normaloff>:/newPrefix/resources/color.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="con_out_size">
          <property name="minimumSize">
           <size>
            <width>16</width>
            <height>16</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>32</width>
            <height>32</height>
           </size>
          </property>
          <property name="statusTip">
           <string>设置译文文字大小</string>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton{border:none;}</string>
          </property>
          <property name="icon">
           <iconset resource="main_ico.qrc">
            <normaloff>:/newPrefix/resources/font.png</normaloff>:/newPrefix/resources/font.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="minimumSize">
           <size>
            <width>60</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="button_clear">
          <property name="text">
           <string>清除</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="actionverson"/>
    <addaction name="actionhelp"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionverson">
   <property name="text">
    <string>版本</string>
   </property>
  </action>
  <action name="actionhelp">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="main_ico.qrc"/>
 </resources>
 <connections/>
</ui>
